src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs

```diff
@@ -18,34 +18,22 @@
                 :number-of-lines 1}
     name]])
 
-(defn profile-info-item [{:keys [label value]}]
-  [react/view
-   [react/view
-    [react/text
-     label]
-    [react/view {:height 10}]
-    [react/touchable-opacity {:on-press #(react/copy-to-clipboard value)}
-    [react/text {:number-of-lines 1
-                 :ellipsizeMode   :middle}
-     value]]]])
-
 (views/defview qr-code []
-  (views/letsubs [current-account [:get-current-account]]
-    (let [public-key (:public-key current-account)]
-      [react/view
-       [react/view {:style styles/close-icon-container}
-        [vector-icons/icon :icons/close {:style styles/close-icon}]]
-       [react/view {:style styles/qr-code-container}
-        [react/text {:style styles/qr-code-title}
-         (string/replace (i18n/label :qr-code-public-key-hint) "\n" "")]
-        [react/view {:style styles/qr-code}
-         [qr-code-viewer/qr-code {:value public-key :size 130} #_{:style styles/qr-code}]]
-        [react/text {:style styles/qr-code-text}
-         public-key]
-        [react/touchable-highlight {:on-press #(react/copy-to-clipboard public-key)}
-         [react/view {:style styles/qr-code-copy}
-          [react/text {:style styles/qr-code-copy-text}
-           (i18n/label :copy-qr)]]]]])))
+  (views/letsubs [{:keys [public-key] :as current-account} [:get-current-account]]
```

Comment:
but you don't use `current-account` why do you need it? just wondering

Reply:
sorry, you're right, fixed it, thought it was referred to in the body of fn.